packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -44,6 +44,7 @@ class DropdownMenuEntry<T> {
   const DropdownMenuEntry({
     required this.value,
     required this.label,
+    this.labelWidget,
```

Comment:
When both `label` and `labelWidget` are specified, which one gets used for the semantics?

Reply:
Good question. The menu entry's MenuItemButton creates a TextButton which includes a container Semantics node, however the real semantics are provided by the labelWidget. So if your labelWidget doesn't include a Text widget or provide Semantics in some other way, then you've got an a11y problem.